app/controllers/posts_controller.rb

```diff
@@ -40,6 +40,26 @@ def destroy
     redirect_to posts_url, notice: 'Post was successfully destroyed.'
   end
 
+  def like
+    @like = @post.likes.build(user: current_user)
```

Comment:
- [ ] The method build is not defined, please 🙏🏽 correct the create method with: 👇🏽 

`@post = @current_user.posts.new(post_params)`